functions/src/import-employees-from-alibeez.ts

```diff
@@ -47,10 +48,14 @@ export const importEmployeesFromAlibeez = async (config: AlibeezConfig) => {
     return;
   }
   const employees: AlibeezEmployee[] = await response.json();
-  const employeesWithValidEmail = employees.filter(
+  const [employeesWithValidEmail, employeesWithNoValidEmail] = partition(
+    employees,
     employee =>
       employee.zenikaEmail && employee.zenikaEmail.endsWith("@zenika.com")
   );
+  employeesWithNoValidEmail.forEach(employee => {
+    console.info("employee with no valid email: " + employee);
```

Comment:
Now that employees are logged individually, I can see more clearly that the code is printing the employees as objects directly. This can be problematic as these objects can 1) become large 2) are not very readable. What could be done instead is only printing the name of the employee.